frontend/src/views/Transactions/components/TransactionView.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import BCBox from '@/components/BCBox';
+import BCTypography from '@/components/BCTypography';
+import { Grid } from '@mui/material';
+import { LabelBox } from './LabelBox'
+
+// Define common inline styles
+const inlineLabelStyle = { display: 'inline', marginRight: 6 };
+
+export const TransactionView = ({ transaction }) => {
```

Comment:
Breadcrumb link for "Admin adjustment" isn't working
<img width="322" alt="image" src="https://github.com/bcgov/lcfs/assets/130073308/eaa6f5b1-85c5-45b6-8709-78737686bb82">

Reply:
I updated our Crumb.jsx component to better handle custom breadcrumbs. 

```Jsx
  // Mapping for custom breadcrumb labels and routes
  const customBreadcrumbs = {
    'admin': { label: 'Administration', route: '/admin' },
    'transfers': { label: 'Transactions', route: '/transactions' },
    'add-org': { label: 'Add organization', route: '/add-org' },
    'edit-org': { label: 'Edit organization', route: '/edit-org' },
    'initiative-agreement': { label: 'Transactions', route: '/transactions' },
    'admin-adjustment': { label: 'Transactions', route: '/transactions' }
  }
  
  const isLast = index === pathnames.length - 1
          const customCrumb = customBreadcrumbs[name] || {}
          const routeTo = customCrumb.route || `/${pathnames.slice(0, index + 1).join('/')}`
          const displayName = customCrumb.label || name.charAt(0).toUpperCase() + name.slice(1).replace('-', ' ')